docs/deployment.md

```diff
@@ -32,16 +32,24 @@ options.
 
 Many instances of collectors can be run in parallel.
 
-## Cassandra
+## Cassandra and ScyllaDB
```

Comment:
I don't feel comfortable claiming the same level of support for Cassandra and ScyllaDB. Cassandra storage has production-level support. ScyllaDB needs to be labeled "Experimental" and with a disclaimer that it has  community support, not the core maintainers support (because we do not have any operational experience with it and will not be able to help with issues).